### PR TITLE
Fix CMakeLists for Android

### PR DESCRIPTION
**Pull request by tjzel**
:shipit:

### CODE REVIEW

File: android/src/main/jni/CMakeLists.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@ target_compile_options(
   -std=c++20
   -Wall
 )
-message(WARNING "CMAKE_CURRENT_SORUCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}")
+message(WARNING "CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}")
```
Comment (WoLewicki) [inline android/src/main/jni/CMakeLists.txt:69]:
```suggestion
message(WARNING "CMAKE_CURRENT_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}")
```

**Comment by tjzel** [comment]
@WoLewicki when will this be merged??? important

**Review by WoLewicki — APPROVED** [review]

**Comment by tjzel** [comment]
@WoLewicki when will it be available in release?

**Comment by Gu1K** [comment]
@WoLewicki up please

**Pull request merged**